src/App.tsx

```diff
@@ -26,70 +27,129 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((item1, item2) => {
+    switch (sortType) {
+      case SortType.LENGTH:
+        return item1.length - item2.length;
+
+      case SortType.ALPHABET:
+        return item1.localeCompare(item2);
+
+      default:
+        return SortType.NONE;
```

Comment:
Be consistent with return values.
```suggestion
        return 0;
```